packages/apollo-server/package.json

```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "apollo-server-core": "^2.0.0-beta.6",
     "apollo-server-express": "^2.0.0-beta.6",
+    "express": "^4.0.0",
```

Comment:
Not really an issue and probably not worth fixing now, but unless we keep the versions between `apollo-server` and `apollo-server-express` in sync this does lead to a Lerna warning:
```
lerna WARN EHOIST_PKG_VERSION "apollo-server-express" package depends on express@4.16.3, which differs from the hoisted express@^4.0.0.
```